scripts/linters/warranted_angular_security_bypasses.py

```diff
@@ -26,5 +26,8 @@
 #  have a longer review time. This is because the reviewers will have to ensure
 #  that the introduction of bypassSecurityTrust was the only option.
 
-EXCLUDED_BYPASS_SECURITY_TRUST_FILES = ()
+EXCLUDED_BYPASS_SECURITY_TRUST_FILES = (
+    'core/templates/components/forms/custom-forms-directives/thumbnail-display.component.spec.ts', # pylint: disable=line-too-long
```

Comment:
This shouldn't be here, try to refactor so that it's not the case.

Reply:
Done!